### PR TITLE
update examples for mmd-3x release

### PR DESCRIPTION
**Pull request by nvijayrania**
# Description
Updating the vision foundation model examples for mmdetection 3.x release

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: cli/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation-pipeline.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ inputs:
   # model_name: microsoft/beit-base-patch16-224
   # # Model - specify the foundation model available in the azureml system registry
   mlflow_model: 
-    path: azureml://registries/azureml/models/mask_rcnn_swin-t-p4-w7_fpn_1x_coco/versions/3
+    path: azureml://registries/azureml/models/mmd-3x-mask-rcnn_swin-t-p4-w7_fpn_1x_coco/versions/8
     type: mlflow_model
```
Comment (rjaincc) [inline cli/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation-pipeline.yaml:29]: Please make changes in cli\foundation-models\system\finetune\image-instance-segmentation\readme.md as well.

File: cli/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-detection-pipeline.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ inputs:
   # model_name: yolof_r50_c5_8x8_1x_coco
   # # model - specify the foundation model available in the azureml system registry
   mlflow_model: 
-    path: azureml://registries/azureml/models/yolof_r50_c5_8x8_1x_coco/versions/3
+    path: azureml://registries/azureml/models/mmd-3x-yolof_r50_c5_8x8_1x_coco/versions/8
     type: mlflow_model
```
Comment (rjaincc) [inline cli/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-detection-pipeline.yaml:30]: Please make changes in cli\foundation-models\system\finetune\image-object-detection\readme.md as well.

File: sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb [hunk 3]
```diff
@@ -265,7 +265,7 @@
    "source": [
     "mmdetection_model_name = \"mask_rcnn_swin-t-p4-w7_fpn_1x_coco\"\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb:266]: the mmdetection model name in 3.x is also different, right?

File: sdk/python/foundation-models/system/import/import_mmdetection_model_into_registry.ipynb [hunk 4]
```diff
@@ -256,7 +256,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "MMDETECTION_MODEL_ID = \"faster_rcnn_r50_fpn_1x_coco\"\n",
+    "MMDETECTION_MODEL_ID = \"faster-rcnn_r50_fpn_1x_coco\"\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/import/import_mmdetection_model_into_registry.ipynb:259]: Please edit the workflow as well

**Review by rjaincc — CHANGES_REQUESTED** [review]
Please take a look at the comments. 
Can you please ensure to update versions of classification models to the latest versions as well in readmes, notebooks and cli?

File: cli/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation-pipeline.yaml (hunk 1 above)
Comment (nvijayrania) [reply cli/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation-pipeline.yaml:29]: Updated. Thanks for catching that

File: cli/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-detection-pipeline.yaml (hunk 2 above)
Comment (nvijayrania) [reply cli/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-detection-pipeline.yaml:30]: Updated.

File: sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb (hunk 3 above)
Comment (nvijayrania) [reply sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb]: Yes, updated to "mask-rcnn_swin-t-p4-w7_fpn_1x_coco"

File: sdk/python/foundation-models/system/import/import_mmdetection_model_into_registry.ipynb (hunk 4 above)
Comment (nvijayrania) [reply sdk/python/foundation-models/system/import/import_mmdetection_model_into_registry.ipynb:259]: Sure. Updated for faster-rcnn.

**Review by sdgilley — APPROVED** [review]
Docs approval

**Review by PhaniShekhar — APPROVED** [review]

**Pull request merged**